crates/relayer-types/src/core/ics03_connection/msgs/conn_open_init.rs

```diff
@@ -180,4 +181,13 @@ mod tests {
         assert_eq!(raw, raw_back);
         assert_eq!(msg, msg_back);
     }
+
+    #[test]
+    fn to_and_from_any() {
+        let raw = get_dummy_raw_msg_conn_open_init();
+        let msg = MsgConnectionOpenInit::try_from(raw.clone()).unwrap();
+        let any = msg.clone().to_any();
```

Comment:
does this any value can be accepted by solidity? I mean directly send this any value to solidity can be ok or not?